src/js/components/Button/doc.js

```diff
@@ -66,7 +66,9 @@ with plain Buttons.`,
 causes the Button to be disabled.`,
     ),
     plain: PropTypes.bool
-      .description('Whether this is a plain button with no border or padding.')
+      .description(
+        'Whether this is a plain button with no border or padding. Defaults to false unless the icon prop exists without a label, or the Button has children.',
```

Comment:
Why would we change the default to true if there is an icon without a label?

Reply:
The logic was always there https://github.com/grommet/grommet/blob/4e309dcd1388662d4e0fe5862b84709749fcaf4f/src/js/components/Button/Button.js#L137
I documented it because I found it confusing until I saw the code and thought it will be better documented.